js/ui/screenShield.js

```diff
@@ -613,17 +624,20 @@ var ScreenShield = class {
         if (this._activationTime == 0)
             this._activationTime = GLib.get_monotonic_time();
 
-        if (Main.sessionMode.currentMode !== 'unlock-dialog' &&
+        if (Main.sessionMode.currentMode !== 'unlock-dialog' ||
```

Comment:
Any current mode is going to be not equal to one or the other of `unlock-dialog` and `unlock-dialog-payg`, so maybe we just don't need this `if`?

Reply:
Good point. The if is entirely useless now. Thanks for pointing that out.